k256/src/schnorr/verifying.rs

```diff
@@ -64,7 +64,6 @@ impl PrehashVerifier<Signature> for VerifyingKey {
         prehash: &[u8],
```

Comment:
Perhaps this should be extracted into something like `verify_raw` and this argument renamed to something which reflects it isn’t necessarily a prehash

Reply:
I extracted out to a new method on `VerifyingKey` with some scary warnings in the rustdoc, ptal